src/js/components/shepherd-content/styles.js

```diff
@@ -4,7 +4,8 @@ export default function contentStyles(variables) {
       background: variables.shepherdTextBackground,
       fontSize: 'inherit',
       outline: 'none',
-      padding: 0
+      padding: 0,
+      borderRadius: variables.shepherdElementBorderRadius
```

Comment:
Can you move this to below `background` please?  Trying to keep them in alphabetical order.

Reply:
Never mind, I will merge and cleanup